docs/source/concept_models/vivarium_us_cvd/concept_model.rst

```diff
@@ -74,235 +74,281 @@ Vivarium - US Health Disparities - CVD Interventions
 
 .. _uscvd2.0:
 
-2.0 Modeling aims and objectives
+2.0 Modeling Aims and Objectives
 ++++++++++++++++++++++++++++++++
 
-The intent of this simulation is to determine the effect of various interventions aimed at: 1) improving blood pressure and LDL-cholesterol control, 2) increasing exercise, 3) decreasing BMI/weight, 4) improving control of fasting plasma glucose, and 5) decreasing tobacco use. 
-
-Initial efforts will focus on the 50 US states and Washington DC; when inputs are available for simulations by race/ethnicity groups within the US subnationals, we will transition to that level. 
+**Objective:** Model the effect of three different interventions on the development of cardiometabolic burden 
+in the United States, contingent on each state’s current population characteristics, patterns of CVD risk, 
+health care access, level of effective medication delivery, and differential effects of risk factor 
+interventions by subgroups. 
 
-Comparisons will be with the GBD results of incidence, prevalence, and mortality for the causes and risks included in the following sections. 
+- Intervetions are aimed at: 1) improving blood pressure and LDL-cholesterol control, 2) increasing exercise, 3) decreasing BMI/weight, and 4) improving control of fasting plasma glucose 
+- Initial efforts will focus on the 50 US states and Washington DC  
+- Comparisons will be with the GBD results of incidence, prevalence, and mortality for various causes and risks associated with CVD  
 
-.. .. _uscvd3.0:
+.. _uscvd3.0:
 
-.. 3.0 Causal framework
-.. ++++++++++++++++++++
+3.0 Concept Model
++++++++++++++++++
 
-.. .. _uscvd3.1:
-
-.. 3.1 Causal diagram
-.. ------------------
- 
-..  .. note::
-..     link to DAGs page
-..     use round circles with DAGs
-
-.. **Outcome (O)**:
-
-
-
-.. **Most proximal determinant/exposure (E)**:
-  
+.. image:: concept_model_v2.svg
 
+.. todo::
+  - Add in legend 
+  - Add minimum viable model diagram 
 
-.. **Confounders (C)**:
+.. _uscvd3.1:
 
+3.1 Simulation Scenarios
+------------------------
 
+.. todo::
+  Lacking clarity on if these are implemented together or one at a time. I am coded them as separate and a final 
+  combined run currently but this is a lot of scenarios to run. Should consider limiting this list. 
 
-.. **Effect modifiers**:
+#. **Baseline**  
+#. **Medical Outreach 50% Coverage**  
+#. **Medical Outreach 100% Coverage**
+#. **Polypill 50% Coverage**  
+#. **Polypill 100% Coverage**
+#. **Lifestyle Modification 50% Coverage**
+#. **Lifestyle Modification 100% Coverage**
+#. **Combination of All Modifications 50% Coverage**
+#. **Combination of All Modifications 100% Coverage**
 
+**Baseline** is assumed to have no one is enrolled in any intervention. 
 
-.. **Mediators (M)**:
+**Medical Outreach 50% Coverage** assumes 50% of eligible simulants are enrolled in the outreach intervention. Scales 
+linearly over 1 year such that there is 0% coverage at baseline and 50% at year 1. 
```

Comment:
Let's add that we want to hold at 50% coverage for the remainder of the sim (add this extension to all scenario descriptions)